### ISSUE TITLE
noramlize,Spectra

### ISSUE THREAD

**Issue by lgatto** [rformassspectrometry/Spectra#199]
We have `normalize,MSnExp` in `MSnbase`, and from my own usage and the [MetaboAnnotation vignette](https://github.com/michaelwitting/MetaboAnnotation/blob/master/vignettes/MetaboAnnotation.Rmd#L242-L245), the use is general. I would suggest we add a `normalise,Spectra` that scales all intensities between 0 and 1.

**Comment by jorainer** [comment]
I think `scale` might be more appropriate (or even `scaleIntensities` to tell what exactly the function is doing).

I think we would also need a parameter to specify the *range* to which scale, you're scaling to [0-1], I'm usually scaling to [0-100] to have percentages and I know there are people that scale to [0-10].

**Comment by lgatto** [comment]
Yes for `scaleIntensitites()` - that will avoid quite a bit of confusion!
